vcpkg/contributing/maintainer-guide.md

```diff
@@ -12,6 +12,29 @@ It is intended to serve the role of
 [Homebrew's Maintainer Guidelines](https://docs.brew.sh/Maintainer-Guidelines), and
 [Homebrew's Formula Cookbook](https://docs.brew.sh/Formula-Cookbook).
 
+## Overall Registry Design Goals

```

Comment:
```suggestion
## Overall registry design goals
```
We should use sentence case for all headings.

Reply:
Also fixed a few other examples that were wron